databricks_cli/stack/api.py

```diff
@@ -294,10 +295,14 @@ def _update_job(self, job_settings, job_id, headers=None):
         :param job_settings: job settings to update the job with.
         :param job_id: physical job_id of job in databricks server.
         """
-
-        self.jobs_client.reset_job({'job_id': job_id, 'new_settings': job_settings},
+        try:
+            self.jobs_client.reset_job({'job_id': job_id, 'new_settings': job_settings},
                                    headers=headers)
-
+        except HTTPError:
+            raise StackError('Job ID could not be found in the workspace while the status file '
+                             'still contains the Job ID {}. Please resolve the inconsistency '
+                             'before proceeding. Aborting stack deployment ...'.format(job_id))
```

Comment:
Nit: Maybe add some suggestions here. For example, `Please remove or modify the local status file...` would be great.

Reply:
Modified

Reply:
Replace "status file" -> "stack status"?
Minor: "Job ID {} in stack status could not be found in the workspace. Please ... "